providers/aws/resources/aws.lr

```diff
@@ -2257,6 +2263,128 @@ aws.ec2 {
   networkAcls() []aws.ec2.networkacl
   // List of keypairs for the account
   keypairs() []aws.ec2.keypair
+  // List of EIPs
+  eips() []aws.ec2.eip
+}
+
+// Amazon Elastic IP
+private aws.ec2.eip {
+  // PublicIP assigned to 
+  publicIp string
+  // false if allocationId not present
+  attached bool 
+  // Ec2 instance associated with the EIP
```

Comment:
I'm not that familiar with AWS. Can an EIP only be attached to an EC2 instance. Not something else?

Reply:
seems like it from looking at the api